index.js

```diff
@@ -97,6 +97,18 @@ module.exports = {
         }
       },
 
+      fetchRevisions: function(context) {
+        var redisDeployClient = this.readConfig('redisDeployClient');
+        var keyPrefix = this.readConfig('keyPrefix');
+
+        this.log('Listing revisions for key: `' + keyPrefix + '`');
+        return Promise.resolve(redisDeployClient.fetchRevisions(keyPrefix))
+          .then(function(revisions){
+            context.revisions = revisions;
```

Comment:
Refer to https://github.com/ghedamat/ember-cli-deploy-redis/blob/add-fetchRevisions/index.js#L72